doc/radosgw/dynamicresharding.rst

```diff
@@ -94,7 +94,7 @@ For example, the output at different Dynamic Resharding stages is shown below:
 
   [
     {
-        "reshard_status": "CLS_RGW_RESHARD_NONE",
+        "reshard_status": "Not resharding",
```

Comment:
It may be better to add a plain-language message rather than replacing an identifier-like token

(Especially for something in JSON where we can have multiple fields.)

Reason being that it would be nice if scripts have an indicator that is robust and unchanging under translation, changes in style, etc.

Reply:
Thank you, to keep the json output machine readable changed the strings
to a middle ground between plain-language and identifier-like output.